dev-tools/openapi-manager/src/output.rs

```diff
@@ -202,31 +217,48 @@ pub(crate) mod headers {
     pub(crate) static GENERATING: &str = "Generating";
 
     pub(crate) static UP_TO_DATE: &str = "Up-to-date";
-    pub(crate) static STALE: &str = "Stale";
-    pub(crate) static MISSING: &str = "Missing";
+    pub(crate) static OUT_OF_DATE: &str = "Out-of-date";
+    pub(crate) static STALE: &str = "-> Stale";
+    pub(crate) static MISSING: &str = "-> Missing";
 
     pub(crate) static UPDATED: &str = "Updated";
     pub(crate) static UNCHANGED: &str = "Unchanged";
 
     pub(crate) static SUCCESS: &str = "Success";
     pub(crate) static FAILURE: &str = "Failure";
 
-    pub(crate) fn continued_indent(count_width: usize) -> String {
+    fn count_section_width(count_width: usize) -> usize {
         // Status strings are of the form:
         //
         //    Generated [ 1/12] api.json: 1 path, 1 schema
+        //             ^^^^^^^^^
         //
-        // So the continued indent is:
-        //
-        // HEADER_WIDTH for the status string
-        // + (count_width * 2) for current and total counts
+        // So the width of the count section is:
+        // (count_width * 2) for current and total counts
         // + 3 for '[/]'
         // + 2 for spaces on either side.
-        " ".repeat(HEADER_WIDTH + count_width * 2 + 3 + 2)
+        count_width * 2 + 3 + 2
+    }
+
+    pub(crate) fn count_section_indent(count_width: usize) -> String {
+        " ".repeat(count_section_width(count_width))
+    }
+
+    pub(crate) fn continued_indent(count_width: usize) -> String {
+        // HEADER_WIDTH for the status string + count_section_width
+        " ".repeat(HEADER_WIDTH + count_section_width(count_width))
     }
 }
 
 pub(crate) mod plural {
```

Comment:
part of me wants to suggest replacing this module with just a function that takes a count and returns either `""` or `"s`, so  you would write `"...  file{}", plural(count))` instead. but, this approach is more resilient to us eventually adding words that pluralize differently, so...it's probably best left as is.

Reply:
Yeah, I've already run into "was"/"were" in a different context so I basically default to this now.